components/icons/README.md

```diff
@@ -58,6 +58,14 @@ To change an icon's color from Tungsten to something else, simply set it from CS
 
 Overriding the size is not recommended. However, if you must, set the `width` and `height` from CSS.
 
+## Accessibility
+
+Icons are generally intended to serve a purely decorative purpose. For icons that do provide some informative context, it may be beneficial to provide an `aria-label` for screen readers:
+
+```html
+<d2l-icon icon="tier3:rubric-graded" role="img" aria-label="Graded rubric icon"></d2l-icon>
+```
+
```

Comment:
Not sure if this is technically a given.. though if it is I'd be fine to just close this down

Reply:
There is some precedent for this:

- [Our own count badge](https://github.com/BrightspaceUI/core/blob/e7559a5410948a1704c8874f36c04e99d2048cf8/components/count-badge/count-badge-icon.js#L113)
- [Insights](https://github.com/Brightspace/insights/blob/3537ed211f79dd8555fd0c4041125ae61ccb9bb2/src/datahub/table.js#L233) (although it maybe should be using the new table column sort stuff)
- [Awards](https://github.com/Brightspace/awards-ui-lit/blob/1e64e2d240333d564ff0256e6c8cea0105cc06aa/src/components/icon/icon.js#L65) (wrapping in a `<div>` but the end result is the same)
- [Consistent Eval](https://github.com/Brightspace/consistent-evaluation/blob/f363f8660c1271cff2a561924aa52ff949903c31/components/left-panel/assignments/consistent-evaluation-assignments-submission-item.js#L482) (missing translations... 😬 )
- A few more that are similar...

Reply:
From my testing (Chrome, Safari, FF using VO):

* `title` on `svg` produces inconsistent results x-browsers; browser tooltip is never displayed
* `aria-label` on `svg` announces fine
* `aria-label` on `d2l-icon` announces fine
* `title` on `d2l-icon` announces fine; browser tooltip is displayed in all browsers

The `role="img"` is needed when labelling `d2l-icon` to avoid announcing the generic "group" role.

Generally, we do not use browser tooltips, and icons + tooltips is a recurring design topic. There is the question of  whether the visual alone is sufficient for sighted non-screenreader users.